ext/standard/mail.c

```diff
@@ -281,6 +320,11 @@ PHPAPI int php_mail(char *to, char *subject, char *message, char *headers, char
 		efree(f);
 	}
 
+	if (hdr && php_mail_detect_multiple_crlf(hdr)) {
+		php_error_docref(NULL TSRMLS_CC, E_WARNING, "Multiple or malformed newlines found in additional_header");
+		MAIL_RET(0);
```

Comment:
Am I understanding correctly that attachment headers should go into the message (body) parameter?

It is not about claiming something, but this is not (clearly) documented:
http://php.net/manual/en/function.mail.php

Looking around for examples I see both methods (using the message and the headers parameters for attachments) being used.

In any case if you are right I see a workaround, but I still believe this change shouldn't break things.